keys/keys.go

```diff
@@ -56,6 +56,12 @@ func NodeStatusKey(nodeID int32) proto.Key {
 	return MakeKey(StatusNodePrefix, encoding.EncodeUvarint(nil, uint64(nodeID)))
 }
 
+// MakeLocalRangeKeyPrefix creates a range-local key prefix from
+// rangeID.
+func MakeLocalRangeKeyPrefix(rangeID proto.RangeID) proto.Key {
```

Comment:
This should be called `MakeRangeIDPrefix`, which is consistent with the other `Prefix`/`Key` method pairs in this file and avoids confusion with `MakeRangeKey` (without the `ID`, which is used for range descriptors and transaction records)